### PR TITLE
[IBCDPE-523] Add UNC collaborators to iAtlas project

### PR DESCRIPTION
**Pull request by BWMac**
This PR adds the UNC collaborators to the iAtlas project for Nextflow Tower Access. It was determined that they do not need direct access to S3 buckets, so this change should be all that is needed. I have not received direct confirmation that their email addresses are connected to Google accounts, but it was communicated to them that they needed to be.

### CODE REVIEW

File: config/projects-prod/iatlas-project.yaml [hunk 1]
```diff
@@ -14,6 +14,12 @@ parameters:
     - "{{stack_group_config.tower_viewer_arn_prefix}}/bruno.grande@sagebase.org"
     - "{{stack_group_config.tower_viewer_arn_prefix}}/adam.taylor@sagebase.org"
     - "{{stack_group_config.tower_viewer_arn_prefix}}/james.eddy@sagebase.org"
+    # The following roles don't exist since the users are not Sage employees.
+    # However, this will ensure that they are added to the Tower workspace
+    # until we find a permanent solution for listing external users.
+    - "{{stack_group_config.tower_viewer_arn_prefix}}/benjamin_vincent@med.unc.edu"
+    - "{{stack_group_config.tower_viewer_arn_prefix}}/dbortone@email.unc.edu"
+    - "{{stack_group_config.tower_viewer_arn_prefix}}/steven_vensko@med.unc.edu"
```
Comment (BrunoGrandePhD) [inline config/projects-prod/iatlas-project.yaml:22]: If these users also need access to the S3 buckets, they will also need to have their actual IAM users/roles added here as well. Also, their Google accounts will need to be added [here](https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/blob/f749c846fb5d59e276a76422ba6cb6c5e3d15932/templates/nextflow-tower-config-bucket.yaml#L72-L74).

Comment (BrunoGrandePhD) [reply config/projects-prod/iatlas-project.yaml:22]: Sorry, I missed that. If these emails are tied to Google accounts, they'll need to be added to the file that I linked.

**Comment by thomasyu888** [comment]
Hold off on this as I want to triple check that these are google accounts.  The information is not provided in the Jira ticket.

**Comment by BWMac** [comment]
@thomasyu888 Are any further changes needed to this PR given the info that these are not google account email addresses?

**Comment by thomasyu888** [comment]
@BWMac , yes - I missed this but we will need to add to this [configuration](https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/blob/f749c846fb5d59e276a76422ba6cb6c5e3d15932/templates/nextflow-tower-config-bucket.yaml#L72-L74).  see Jira ticket for more information

**Review by thomasyu888 — APPROVED** [review]
🚀  LGTM!

**Pull request merged**